packages/renderer/src/stores/kubernetes-contexts.ts

```diff
@@ -0,0 +1,61 @@
+/**********************************************************************
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import { writable, type Writable } from 'svelte/store';
+import { EventStore } from './event-store';
+import { getKubeUIContexts, type KubeContextUI } from '../lib/kube/KubeContextUI';
+
+const windowEvents: string[] = [
+  'extensions-started',
+  'kubernetes-context-update',
+  'kubernetes-context-create',
+  'kubernetes-context-delete',
```

Comment:
if we have only one event it looks fine no ?

Reply:
Changed to only check kubeneretes-context-update! Thanks for pointing that out.